crates/clorinde/src/codegen/cargo.rs

```diff
@@ -43,253 +40,208 @@ impl DependencyAnalysis {
 }
 
 pub fn gen_cargo_file(dependency_analysis: &DependencyAnalysis, config: &Config) -> String {
+    let mut deps_toml = String::new();
     let package = config
         .package
         .to_string()
         .expect("unable to serialize package");
-
-    let mut buf = formatdoc! {r#"
-        # This file was generated with `clorinde`. Do not modify
-        {package}
-    "#};
-
-    if config.r#async {
-        let mut default_features = vec!["\"deadpool\""];
-        if dependency_analysis.has_dependency() && dependency_analysis.chrono {
-            default_features.push("\"chrono\"");
-        }
-
-        let mut wasm_features = vec!["\"tokio-postgres/js\""];
-        if dependency_analysis.has_dependency() && dependency_analysis.chrono {
-            wasm_features.push("\"chrono?/wasmbind\"");
-            wasm_features.push("\"time?/wasm-bindgen\"");
-        }
-
-        let default_features = default_features.join(", ");
-        let wasm_features = wasm_features.join(", ");
-
-        writedoc! { buf, r#"
-
-            [features]
-            default = [{default_features}]
-            deadpool = ["dep:deadpool-postgres", "tokio-postgres/default"]
-            wasm-async = [{wasm_features}]
-        "#}
-        .unwrap()
-    } else {
-        let mut wasm_features = vec![];
-        if dependency_analysis.has_dependency() && dependency_analysis.chrono {
-            wasm_features.push("\"chrono?/wasmbind\"");
-        }
-
-        let wasm_features = wasm_features.join(", ");
-
-        writedoc! { buf, r#"
-
-            [features]
-            default = []
-            wasm-sync = [{wasm_features}]
-        "#}
-        .unwrap();
-    }
-
-    if dependency_analysis.chrono {
-        writedoc! { buf, r#"
-
-            chrono = ["dep:chrono"]
-            time = ["dep:time"]
-        "#}
-        .unwrap();
-    } else {
-        writedoc! { buf, r#"
-
-            chrono = []
-            time = []
-        "#}
-        .unwrap();
-    }
-
-    writedoc! { buf, r#"
-
-        [dependencies]
-        ## Core dependencies
-        # Postgres types
-        postgres-types = {{ version = "0.2.8", features = ["derive"] }}
-        # Postgres interaction
-        postgres-protocol = "0.6.7"
-        # Iterator utils required for working with `postgres_protocol::types::ArrayValues`
-        fallible-iterator = "0.2.0"
-    "#}
-    .unwrap();
-
-    // add custom type crates
-    if !config.types.mapping.is_empty() {
```

Comment:
issue(blocking): this entire feature seems to be gone, and I don't see a replacement.